### ISSUE TITLE
Incorrect result?

### ISSUE THREAD

**Issue by baggepinnen** [daschw/DirectDependents.jl#1]
The following
```julia
julia> direct_dependents("Stan")
2-element Array{String,1}:
 "DiffEqBayes"
 "Turing"
```
does not appear to be correct, Turing does not depend on Stan 
https://github.com/TuringLang/Turing.jl/blob/master/Project.toml
Does this package perhaps consider all possible registered versions of Turing and one of those might have had a dependency on Stan.jl?

**Comment by daschw** [comment]
Yes, that's exactly the issue. Versions are not considered at all here. This could probably be extended quite easily.
Thanks for the hint!
